cmd/gateway/s3x/s3.proto

```diff
@@ -114,8 +107,8 @@ message ObjectPartInfo {
 }
 
 message MultipartUpload {
-    string bucket = 1;
-    string object = 2;
+    ObjectInfo objectInfo = 1;
     string id = 3;
-    repeated ObjectPartInfo objectParts = 4 [(gogoproto.nullable) = false];
+    //map of index to parts
+    map<int64, ObjectPartInfo>  objectParts = 4 [(gogoproto.nullable) = false];
```

Comment:
this should be `map<int64, ObjectPartInfo>  objectParts = 3 [(gogoproto.nullable) = false];` now that the `object` field was removed

Reply:
got it